sass/directives/_accents.scss

```diff
@@ -47,3 +47,54 @@
   width: 100%;
   left: 0;
 }
+
+@mixin return-to-top {
+  position: fixed;
+  bottom: 1.25em;
+  right: 1.25em;
+  background: $grey-darkest;
+  opacity: 0.6;
+  width: 3.125em;
+  height: 3.125em;
+  display: block;
+  text-decoration: none;
+  -webkit-border-radius: 35px;
+  -moz-border-radius: 35px;
+  border-radius: 35px;
+  display: none;
+  -webkit-transition: all 0.3s linear;
+  -moz-transition: all 0.3s ease;
+  -ms-transition: all 0.3s ease;
+  -o-transition: all 0.3s ease;
+  transition: all 0.3s ease;
+  z-index: 99999999999999;
```

Comment:
I'm not sure I'd specify a default `z-index` like this; instead, I'd force the consuming application to come up with a working number